build/bin/upgrade.sh

```diff
@@ -20,7 +20,7 @@
 function help() {
```

Comment:
This file can be removed, it is useless now.

Reply:
Done. upgrade.sh has been removed.